src/components/UserAccess/UserAccessForm/form-utils.ts

```diff
@@ -56,7 +56,7 @@ export const createRBs = async (
     metadata: {
       // To sanitize the username and ensure every user just has one role
       // in the namespace.
-      name: `${sanitizeUsername(username)}`,
+      name: `konflux-${role.toLowerCase()}-${sanitizeUsername(username)}-user-actions`,
```

Comment:
Should we store this `-user-actions` in const like USER_ACTION_SUFFIX?

Reply:
How we avoid 'one user has multiple accounts'? Let the doc team improve this part>